src/OutsideClick.js

```diff
@@ -33,6 +33,8 @@ class OutsideAlerter extends Component {
     if (this.wrapperRef && !this.wrapperRef.contains(event.target)) {
       if(event.target.className === 'page-mask'){
         this.props.toggleModalPost(false);
+        this.props.toggleModalSignIn(false);
```

Comment:
Do you need to emit 3 different actions? Or can you simplify this to 1?

Reply:
I simplified the action function, but cannot simplify this to 1, because I need to make sure all three modals are turn off when the page mask is clicked on.